src/components/draw/feature-item.vue

```diff
@@ -56,9 +56,11 @@ function onClickDelete() {
     <span
       class="lux-icon"
       :class="{
-        point: feature.label === 'Point',
-        line: feature.label === 'LineString',
-        polygon: feature.label === 'Polygon' || feature.label === 'Circle',
+        point: feature.featureType === 'drawnPoint',
```

Comment:
icon for label is missing because of these new conditions
![image](https://github.com/user-attachments/assets/18cc5b6d-5e99-4aa9-a6fe-1fb5377f7173)

Reply:
There is no icon for Labels

Reply:
I put a condition to display a point for a label, but it would be better to create an icon for the label